librz/core/cdebug.c

```diff
@@ -642,3 +642,38 @@ RZ_API void rz_debug_traces_ascii(RzDebug *dbg, ut64 offset) {
 	rz_table_free(table);
 	rz_list_free(info_list);
 }
+
+/**
+ * \brief close debug process
```

Comment:
```suggestion
 * \brief Close debug process
```
What does it mean to "close" the debug process? Probably a bit more descriptive description would be great in this case.